covid_act_now/tests/test_geo.py

```diff
@@ -0,0 +1,115 @@
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from delphi_covid_act_now.geo import (
+    positivity_rate,
+    std_err,
+    geo_map
+)
+
+class TestAggregationFunctions:
+    def test_pos_rate(self):
+        df = pd.DataFrame({
+            "pcr_tests_positive": [0, 1, 2, 3, 4, 5],
+            "sample_size": [2, 2, 5, 10, 20, 50]
+        })
+
+        # The 0 sample_size case is expected to return 0 following the CDC's convention
+        expected_pos_rate = [0, 0.5, 0.4, 0.3, 0.2, 0.1]
+        pos_rate = positivity_rate(df)
+
+        assert np.allclose(pos_rate, expected_pos_rate)
+
+    def test_std_err(self):
+        df = pd.DataFrame({
+            "val": [0, 0.5, 0.4, 0.3, 0.2, 0.1],
+            "sample_size": [2, 2, 5, 10, 20, 50]
+        })
+
+        expected_se = np.sqrt(df.val * (1 - df.val) / df.sample_size)
+        se = std_err(df)
+
+        assert (se >= 0).all()
```

Comment:
```suggestion
        # 0 se is permitted in this indicator, since applying the Jeffreys prior would violate the mirror
        assert (se >= 0).all()
```